salt/modules/vmctl.py

```diff
@@ -0,0 +1,378 @@
+# -*- coding: utf-8 -*-
+# $OpenBSD$
+'''
+Manage vms running on the OpenBSD VMM hypervisor using vmctl(8).
+
+.. note::
+
+    This module requires the `vmd` service to be running on the OpenBSD
+    target machine.
+'''
+
+from __future__ import absolute_import
+
+# Import python libs
+import logging
+import re
+
+# Imoprt salt libs:
+import salt.utils
```

Comment:
We're moving away from bare `salt.utils` imports. All of the functions there have been moved over into their own files. Can you update this import to import the individual files needed `import salt.utils.x`? (And of course update the references in the function calls below.)

The `salt.utils.__init__.py` file should have the up-to-date paths in the deprecation warnings for each function for reference.

Reply:
There was actually only one (`salt.utils.path`). I'll adjust it.